### PR TITLE
Fix the build after changes in LLVM

### PR DESCRIPTION
**Pull request by AlexeySotkin**
The following commits broke the build:
https://github.com/llvm/llvm-project/commit/05da2fe52162c80dfa18aedf70cf73cb11201811
https://github.com/llvm/llvm-project/commit/631be5c0d41161057d02fe08a7aeb4fbde1a91d6
https://github.com/llvm/llvm-project/commit/0908093977b2b80d00baa12f0b2f1424dde096fb

Signed-off-by: Alexey Sotkin <alexey.sotkin@intel.com>

### CODE REVIEW

**Comment by airlied** [comment]
I'm also seeing a need to add the initializepasses include to SPIRVWriter.cpp

File: lib/SPIRV/SPIRVLowerSPIRBlocks.cpp [hunk 1]
```diff
@@ -45,6 +45,7 @@
 #include "llvm/Analysis/AliasAnalysis.h"
 #include "llvm/Analysis/AssumptionCache.h"
 #include "llvm/Analysis/CallGraph.h"
+#include "llvm/InitializePasses.h"
```
Comment (svenvh) [inline lib/SPIRV/SPIRVLowerSPIRBlocks.cpp]: Please move this after `#include "llvm/IR/Verifier.h"`

**Comment by svenvh** [comment]
> I'm also seeing a need to add the initializepasses include to SPIRVWriter.cpp

@airlied you mean you are still seeing build failures?  This patch as is fixes the build for me against LLVM `6ec0714098`.

**Comment by airlied** [comment]
@svenvh yes when I was building it yesterday against 6a1b51282bc5728b77a80581a4aa38edcc9f95e5
I had to add

#include "llvm/InitializePasses.h"
to lib/SPIRV/SPIRVWriter.cpp as well

File: lib/SPIRV/SPIRVLowerSPIRBlocks.cpp (hunk 1 above)
Comment (AlexeySotkin) [reply lib/SPIRV/SPIRVLowerSPIRBlocks.cpp]: Done

**Comment by AlexeySotkin** [comment]
@airlied, to me it builds fine without changes in lib/SPIRV/SPIRVWriter.cpp. And it seems travis can build it too. Are you sure the changes in lib/SPIRV/SPIRVWriter.cpp are required?

**Comment by AlexeySotkin** [comment]
> There seem to be other issues that I'm looking at (https://reviews.llvm.org/D69416), but if you fix the include file order I'm okay to merge this.

I agree, we can merge this PR now and travis ci should be fixed automatically, once https://reviews.llvm.org/D69416 gets to the package.

**Pull request merged**